cdap-hbase-compat-0.98/src/main/java/co/cask/cdap/data2/util/hbase/HBase98TableUtil.java

```diff
@@ -257,21 +259,29 @@ public BloomType getBloomFilter(HColumnDescriptor columnDescriptor) {
   }
 
   @Override
-  public Map<String, TableStats> getTableStats(HBaseAdmin admin) throws IOException {
+  public Map<TableId, TableStats> getTableStats(CConfiguration conf, HBaseAdmin admin) throws IOException {
     // The idea is to walk thru live region servers, collect table region stats and aggregate them towards table total
     // metrics.
-    Map<String, TableStats> datasetStat = Maps.newHashMap();
+    String root = conf.get(Constants.Dataset.TABLE_PREFIX);
+    Map<TableId, TableStats> datasetStat = Maps.newHashMap();
     ClusterStatus clusterStatus = admin.getClusterStatus();
 
     for (ServerName serverName : clusterStatus.getServers()) {
       Map<byte[], RegionLoad> regionsLoad = clusterStatus.getLoad(serverName).getRegionsLoad();
 
       for (RegionLoad regionLoad : regionsLoad.values()) {
-        String tableName = Bytes.toString(HRegionInfo.getTableName(regionLoad.getName()));
+        //String tableName = Bytes.toString(HRegionInfo.getTableName(regionLoad.getName()));
+        TableName tableName = HRegionInfo.getTable(regionLoad.getName());
+        String namespace = tableName.getNamespaceAsString();
+        String qualifier = tableName.getQualifierAsString();
+        if (!namespace.startsWith(root + "_") &&
```

Comment:
Look into the isCDAPTable method.
That may be what you want.

(Note that that is a little more liberal than this, since I don't check the namespace there).